lib/testrunner

```diff
@@ -14,7 +16,9 @@ KERNEL_ARCH=""
 ktest_dir="/host/$ktest_dir"
 ktest_tmp="/host/$ktest_tmp"
 
-ln -s $ktest_dir /ktest
+if [ ! -e "/ktest" ]; then
+    ln -s $ktest_dir /ktest
+fi
```

Comment:
why make this a conditional? we're always running in a brand new vm here

Reply:
There are some reboot tests(ex: backingdev_reboot_fio.ktest, backingdev_reboot_fs.ktest, ...), and the directory of ktest have been linked. So do link again would return error and break the test after next reboot.